src/main/java/frc/robot/Constants.java

```diff
@@ -74,4 +74,49 @@ public static class Intake {
     /** Speed we want to run the Intake at. */
     public static final double INTAKE_MOTOR_SPEED = 0.5;
   }
+
+  /** Holds contstants for the limelights. */
+  public static class Limelight {
+    /** Constants for aiming limelight. */
+    public static class AimingLimelight {
+      /** The number of degrees the limelight is mounted back from perfectly vertical */
+      public static final double MOUNT_ANGLE = 0;
+
+      /** The number of inches from the center of the Limelight lens to the floor */
+      public static final double MOUNT_HEIGHT = 0;
+
+      /** The number of inches from the retroreflective tape on the upper hub to the floor */
+      /**
```

Comment:
Fix this JavaDoc to actually describe what the var is. Also you can't have multiple JavaDoc comments on one field I don't think